t/harness

```diff
@@ -292,71 +335,71 @@ my $h = TAP::Harness->new({
     verbosity   => $Verbose,
     timer       => $ENV{HARNESS_TIMER},
     exec        => sub {
-	my ($harness, $test) = @_;
```

Comment:
isnt our policy to replace tabs with spaces?

Reply:
rebase error